Description: Fix build failure with [-Werror=format-security]
Author: Michael Biebl <biebl@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=643423

Let chunk stripped due to Gentoo revert-forkpty patch.

Index: libgksu-2.0.13~pre1/libgksu/libgksu.c
===================================================================
--- libgksu-2.0.13~pre1.orig/libgksu/libgksu.c	2011-12-07 11:01:18.351654566 +0100
+++ libgksu-2.0.13~pre1/libgksu/libgksu.c	2011-12-07 11:01:41.231654725 +0100
@@ -1276,7 +1276,7 @@
   context->dir = g_strdup (mkdtemp(template));
   if (!context->dir)
     {
-      fprintf (stderr, strerror(errno));
+      fprintf (stderr, "%s", strerror(errno));
       return FALSE;
     }
 
@@ -2247,7 +2247,7 @@
 					  " %s"), converted_str, "gksu: waiting");
 	      g_free (converted_str);
 
-	      g_set_error (error, gksu_quark, GKSU_ERROR_HELPER, emsg);
+	      g_set_error_literal (error, gksu_quark, GKSU_ERROR_HELPER, emsg);
 	      g_free (emsg);
 
 	      if (context->debug)
